"""
了解异常
捕获异常
异常的else
异常的finally
异常的传递
自定义异常
"""
"""
当检测到一个错误时，解释器就无法继续执行了，反而出现了一些错位提示，这就是所谓的”异常“。
"""

"""
语法：
    try:
        可能发生错误的代码
    except:
        如果出现异常执行的代码
"""
try:
    f = open('test.txt', 'r')
except:
    f = open('test.txt', 'w')
